### PR TITLE
Add prometheus annotations for filebeat metrics service

### PR DESCRIPTION
**Pull request by pastukhov**
Signed-off-by: Artem Pastukhov <artem.pastukhov@gmail.com>

Add prometheus annotations for filebeat metrics service
For those who have no priometheus operator

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/filebeat/templates/service.yaml [hunk 1]
```diff
@@ -2,6 +2,10 @@
 kind: Service
 apiVersion: v1
 metadata:
+  annotations:
+    prometheus.io/path: /metrics
+    prometheus.io/port: "9479"
+    prometheus.io/scrape: "true"
```
Comment (sstarcher) [inline stable/filebeat/templates/service.yaml:13]: We should likely not have these when the serviceMonitor is disabled - https://github.com/helm/charts/blob/master/stable/filebeat/values.yaml#L150

**Review by sstarcher — CHANGES_REQUESTED** [review]

**Comment by monotek** [comment]
Could you also raise the image version to 6.6.1 please?

File: stable/filebeat/templates/service.yaml (hunk 1 above)
Comment (pastukhov) [reply stable/filebeat/templates/service.yaml:13]: Done

**Pull request merged**